src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1204,6 +1208,17 @@ static Library()
                 BuiltinFunctionsCore.Pi,
                 NoErrorHandling(Pi)
             },
+            {
+                BuiltinFunctionsCore.PlainText,
+                StandardErrorHandling<FormulaValue>(
+                    BuiltinFunctionsCore.PlainText.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: NoOpAlreadyHandledByIR,
+                    checkRuntimeTypes: DeferRuntimeTypeChecking,
```

Comment:
>DeferRuntimeTypeChecking [](http://example.com/codeflow?start=39&length=24)

Should this be `ExactValueTypeOrBlank<StringValue>` ?

Reply:
No. want the value is formulavalue.

Reply:
I think Line 1213 Also needs to be `StringValue`

Reply:
updated.